jitbuilder/release/cpp/samples/OperandStackTests.cpp

```diff
@@ -585,7 +585,8 @@ OperandStackTestUsingStructMethod::OperandStackTestUsingStructMethod(OMR::JitBui
    : OperandStackTestMethod(d)
    {
    d->DefineStruct("Thread");
-   d->DefineField("Thread", "sp", d->PointerTo(d->PointerTo(STACKVALUEILTYPE)), offsetof(Thread, sp));
+   //d->DefineField("Thread", "sp", d->PointerTo(d->PointerTo(STACKVALUEILTYPE)), offsetof(Thread, sp));
```

Comment:
Can you remove the commented out line?

Reply:
bah, yes of course....that was poor cleanup on my part :(